WBooks/Screens/Library/LibraryCell/LibraryCellView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  CellView.swift
+//  WBooks
+//
+//  Created by Ariel Cid on 3/15/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Core
+
+class LibraryCellView: UITableViewCell {
+    
+    @IBOutlet weak var bookCover: UIImageView!
+    @IBOutlet weak var bookTitle: UILabel!
+    @IBOutlet weak var bookAuthor: UILabel!
+    
+    var imageFetchingTask: URLSessionDataTask? = .none
+    
+    override internal func awakeFromNib() {
+        super.awakeFromNib()
+        
+        setupUI()
+    }
+    
+    func setCellData(bookVM: BookViewModel) {
+        bookTitle.text = bookVM.title
+        bookAuthor.text = bookVM.author
+        configureImage(urlBook: bookVM.image)
+    }
+    
+    override func prepareForReuse() {
+        imageFetchingTask?.cancel()
+        imageFetchingTask = .none
+    }
+    
+    func configureImage(urlBook: URL?) {
+        guard let url = urlBook else { return }
+        imageFetchingTask = URLSession.shared.dataTask(with: url) { (data, response, error) in
```

Comment:
Can you update this code and use Utils? (you can copy paste from Utils)

Reply:
https://github.com/Wolox/wolmo-utils-ios/blob/master/Utils/Services/ImageFetcher.swift

Reply:
And this should be in `BookViewModel` probably you may do a function like that:
`func downloadImage(closure: @escaping (UIImage) -> Void) {`

Reply:
I was already working on this, and got it to work. The only thing left is that func you just mentioned.